README.md

```diff
@@ -23,37 +23,37 @@ Welcome to **EasyApp**, your one-stop solution for navigating job listings with
 - **Navigate Listings**: Manage and navigate through your job listings.
 - **Settings**: Customize your personal info for EasyApp to use.
 
+## Contributing
+
+Anyone is allowed to contribute to EasyApp. The guidelines for contribution can be found in the [Contributing](Contributing) file.
+
 ## Getting Started
 
 ### Installation
 
 1. Clone this repository:
 
-```bash
-git clone https://github.com/EasyApp-RPI/EasyApp.git
-```
+    ```bash
+    git clone https://github.com/EasyApp-RPI/EasyApp.git
+    ```
 
-2. Open the repository on your machine nd install node packages with:
+2. Open the repository on your machine and install node packages with:
 
-```bash
-npm install
-```
+    ```bash
+    npm install
+    ```
 
 3. Then build the chrome extension with:
 
-```bash
-npm run build
-```
+    ```bash
+    npm run build
+    ```
 
 4. Open Chrome and navigate to `chrome://extensions/`.
 5. Enable "Developer mode" (usually a toggle in the top right).
 6. Click on "Load unpacked" and select the `EasyApp/extension` directory.
 7. The **EasyApp** extension icon should now appear in your Chrome toolbar. Click on it and enjoy!
 
-### For Devs
-
-If you alter the react chrome extension or autofill script at all make sure to run `npm run build`. If you alter the extension permissions in the manifest make sure to reload the extension.
-
```

Comment:
I feel like this should be emphasized somewhere, but this might not the be best place?

Reply:
It makes the most sense to put it in the contribution guide imo. Its where we go over setting up the dev build environment and the general workflow of the project. If someone actually wants to contribute to the project they will most likely read the contribution guide. Those not interested in contributing will just read the readme for installation.